src/TenancyServiceProvider.php

```diff
@@ -156,4 +145,11 @@ public function boot(): void
             return $instance;
         });
     }
+
+    public function provides()
+    {
+        return [
+            'command.migrate.fresh'
+        ];
+    }
```

Comment:
What's the point of this?

Reply:
I saw that the guy with the `make:model` override package had it in the package's provider. I tried adding it to see if that would help with overriding the command, but it didn't help. I couldn't find documentation of the method, so honestly, I don't know what the method does

Reply:
Just found the documentation and learned about deferred providers (the `provides` method is only needed in deferred providers) https://laravel.com/docs/9.x/providers#deferred-providers